src/components/Match/MatchStory.jsx

```diff
@@ -540,6 +557,18 @@ const generateStory = (match) => {
   // Intro
   events.push(new IntroEvent(match));
 
+  // Prediction
+  let predExists = false;
+  match.players.forEach((player) => {
+    if (player.pred_vict === true) {
+      predExists = true;
+    }
+  });
+  if (predExists === true) {
+    events.push(new PredictionEvent(match, -89));
+    events.push(new PredictionEvent(match, -88));
```

Comment:
Looks like this will add a prediction event even if nobody on that team predicted, as long as the other team did. Example match: 3159888768.